bench_runner/__main__.py

```diff
@@ -36,5 +47,5 @@
         sys.exit(1)
 
     sys.argv = [sys.argv[0], *sys.argv[2:]]
-    mod = importlib.import_module(f"bench_runner.scripts.{command}")
+    mod = importlib.import_module(f".{command}", "bench_runner.scripts")
```

Comment:
This is just due to picking the unfortunate name `bisect` that conflicts with the name of a stdlib module and not being careful about how things are run elsewhere.